clusterHost.py

```diff
@@ -198,20 +199,19 @@ def _preinstall() -> host.Result:
 
         return executor.submit(_preinstall)
 
-    def _start_nodes(self, iso_path: str, infra_env: str, executor: ThreadPoolExecutor, nodes: list[ClusterNode]) -> list[Future[Optional[host.Result]]]:
-        self._ensure_images(iso_path, infra_env, nodes)
```

Comment:
After this commit, we don't do ensure_images anymore. Can we split up the commit? I know that the final commit is the most interesting one, but if we split it up, then we get all the other preparation commits in first.